CMakeLists.txt

```diff
@@ -10,6 +13,17 @@ include(meshioChecks)
 option(MeshIO_BUILD_DOCS "Build MeshIO Documentation" OFF)
 option(MeshIO_BUILD_EXAMPLES "Build MeshIO Examples" OFF)
 option(MeshIO_BUILD_COVERAGE "Generate MeshIO coverage report" OFF)
+option(MeshIO_BUILD_TESTS "Build unit tests" OFF)
+
+if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
+  # Ensure -std=c++xx instead of -std=g++xx
+  set(CMAKE_CXX_EXTENSIONS OFF)
+
+  # Nicely support folders in IDEs
+  set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+  include(CTest)
+endif()
```

Comment:
Please remove `include(CTest)` here. It is only needed when in the next IF block you added.

Reply:
Done.